src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs

```diff
@@ -93,8 +93,21 @@ protected override void FreeNotFirstVisitor(AbstractSymbolDisplayVisitor visitor
 
         internal SymbolDisplayPart CreatePart(SymbolDisplayPartKind kind, ISymbol? symbol, string text)
         {
-            text = (text == null) ? "?" :
-                   (_escapeKeywordIdentifiers && IsEscapable(kind)) ? EscapeIdentifier(text) : text;
+            if (text == null)
+            {
+                return new SymbolDisplayPart(kind, symbol, "?");
+            }
+
+            if (!_escapeKeywordIdentifiers)
+            {
+                return new SymbolDisplayPart(kind, symbol, text);
+            }
+
+            bool isNamedTypeOrAliasName = symbol?.Kind is SymbolKind.NamedType or SymbolKind.Alias;
```

Comment:
>SymbolKind.Alias [](http://example.com/codeflow?start=82&length=16)

Are we testing the alias case? (See `SymbolDisplayTests.TestAlias1` for an example of an alias test.)

Reply:
Thanks,  I have added a test for an alias. If `Goo` the new `Foo`? :)